src/routes/routes.js

```diff
@@ -11,29 +11,30 @@ import DashboardView from '../components/dashboard/dashboard-view';
 import UpTimeView from '../components/uptime/uptime-view';
 import MonitorView from '../components/monitor/monitor-view';
 import IntegrationView from '../components/integrations/integrations-view';
-
-// const PrivateRoute = ({ component: Component, ...rest }) => (
-//   <Route {...rest} render={(props) => (
-//     isUserActive() ? <Component {...props} /> : <Redirect to={getRedirectionUrl({...rest}, props)} />
-//   )} />
-// );
+import { isUserActive } from '../helpers/auth-helpers';
 
 const PrivateRoute = ({ component: Component, ...rest }) => (
   <Route {...rest} render={(props) => (
-    true ? <Component {...props} /> : <Redirect to='/' />
+    isUserActive() ? <Component {...props} /> : <Redirect to="/" />
+  )} />
+);
+
+const PublicRoute = ({ component: Component, ...rest }) => (
+  <Route {...rest} render={(props) => (
+    isUserActive() ? <Redirect to="/dashboard" /> : <Component {...props} />
```

Comment:
Do we need to check isUserActive for public routes?

Reply:
Yes. So say there is a use case when user is active and on `/dashboard` url and he/she tries to access public urls like `/`, `/register` or `/forgot-password` in that case we should check the active session and redirect him/her to default route i.e. `/dashboard`.

Reply:
i don't think the user would be entering this constant in that case. since you have already defined /forget-password as a public route it'll always go there..

Reply:
We can not restrict a user from changing URLs manually. So this redirection will take care in such cases when a user tries to change the URLs manually.